kernel/nativecode.ml

```diff
@@ -2130,11 +2125,8 @@ let compile_deps env sigma prefix ~interactive init t =
   in
   aux env 0 init t
 
-let compile_constant_field env prefix con acc cb =
-    let (gl, _) =
-      compile_constant ~interactive:false env empty_evars prefix
-        con cb
-    in
+let compile_constant_field env _prefix con acc cb =
```

Comment:
Note that I could also get rid of `_prefix` here but it's becoming slightly unrelated with this PR.

Reply:
@ppedrot I await your PR